doc/nrf/ug_nrf_cloud.rst

```diff
@@ -56,6 +56,42 @@ MQTT overview
 * Once connected, the device subscribes to the desired MQTT topics.
 * Each MQTT publish event contains the MQTT topic and the payload.
 
+Security
+********
+
+A device can successfully connect to `nRF Cloud`_ using REST if:
+
+* the device contains a correct x509 CA certificate, and private key
+* the public key derived from the private key is registered with an nRF Cloud account
+* the device calls nRF Cloud REST APIs which require a JSON Web Token (JWT) by providing a JWT signed by the private key
```

Comment:
```suggestion
* The device contains a correct x509 CA certificate, and private key.
* The public key derived from the private key is registered with an nRF Cloud account.
* The device calls nRF Cloud REST APIs which require a JSON Web Token (JWT) by providing a JWT signed by the private key
```
The last item does not read very well (thinking of it as a requirement).
Maybe something like: " A JSON Web Token (JWT) signed by the private key is available for the nRF Cloud REST API calls."?